pkg/rtc/participant.go

```diff
@@ -391,7 +391,7 @@ func (p *ParticipantImpl) SetPermission(permission *livekit.ParticipantPermissio
 		p.SubscriptionManager.queueReconcile("")
 	} else {
 		// revoke all subscriptions
-		for _, st := range p.GetSubscribedTracks() {
+		for _, st := range p.SubscriptionManager.GetSubscribedTracks() {
```

Comment:
Just making use of this consistent. It is an embedded structure, so don't need the explicit use, but I see a couple of places which does the explicit call.